spec/features/in_person_doc_auth_bug_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'recreating the LG-13472 bug' do
```

Comment:
seems like this is in the handoff step, so would `spec/features/idv/doc_auth/hybrid_handoff_spec.rb` be a good home for this spec?

Reply:
I have no plans to actually merge this change. This is here so that CI can run and I can demonstrate how to reproduce it.